src/Menu/Examples.md

```diff
@@ -4,29 +4,29 @@
 intialState = { isOpen: false };
 
 function toggleOpen() {
-  setState({ isOpen: !state.isOpen })
+  setState({ isOpen: !state.isOpen });
```

Comment:
are these changes ok? are we formatting the other examples?

Reply:
actually i didn't notice this, but looks compliant with the standard formatting we use everywhere.
Didn't it?

Reply:
Examples are using inconsistent formatting. I'm ok with it.

Reply:
Yes i just verified and indeed there is an overall inconsistency in examples, but I think this should be discussed in a separate issue.